packages/syntax-icons/src/generate-icons/iconTemplate.tsx

```diff
@@ -0,0 +1,20 @@
+import { type Template } from "@svgr/babel-plugin-transform-svg-component";
+
+const iconTemplate: Template = ({ componentName, jsx }, { tpl }) => {
+  return tpl`
+  import { type ComponentProps, forwardRef } from "react";
+  import Icon from "../../../syntax-core/src/Icon/Icon";
+
+  const ${componentName} = forwardRef<
+    SVGSVGElement,
+    Omit<ComponentProps<typeof Icon>, "path">
+  >(({ color, size }, ref) => (
+    <Icon ref={ref} path="${jsx.children[0].openingElement.attributes[0].value.value}" color={color} size={size} />
+  ));
+
+  ${componentName}.displayName = "${componentName}";
+  export default ${componentName}
+  `;
+};
+
+export default iconTemplate;
```

Comment:
@somethiiing would be great if we can use `export default function iconTemplate` instead of using `const` to define the function.

Reply:
hm, this looks a little harder to read compared to the first variation:

![image](https://github.com/Cambly/syntax/assets/15243713/cb327660-67b8-45a2-9cac-9e78f6aaeb9b)

i'm going to leave it as the original unless you feel more strongly about it